src/components/fx/hover.js

```diff
@@ -1922,20 +1922,28 @@ function alignHoverText(hoverLabels, rotateLabels, scaleX, scaleY) {
         var offsetY = offsets.y;
 
         var isMiddle = anchor === 'middle';
+        var showArrow = d.trace.hoverlabel?.showarrow;
```

Comment:
Should `undefined` result in a value of true?
```suggestion
        var showArrow = 'showarrow' in d.trace.hoverlabel : d.trace.hoverlabel.showarrow : true;
```

Reply:
@camdecoster Yes, at the moment this is handled by the `showArrow !== false` check on line 1932 but I suppose it's clearer to do it this way, will update.

The actual issue that `d.trace.hoverlabel` itself may be undefined (since it is not supported for some traces). If `d.trace.hoverlabel` is defined, `d.trace.hoverlabel` will already be assigned `true` if a value is given by the user.